internal/modules/oracle/test.py

```diff
@@ -78,28 +79,28 @@ def test_query_oracle_params(self):
     # test_prevotes tests to make sure that we can submit prevotes
     def test_prevotes(self):
         # Get Hash
-        vote_hash_1 = get_hash(EXCHANGE_RATES.ToString(), STATIC_SALT, validator1_acc["address"])
-        vote_hash_2 = get_hash(EXCHANGE_RATES.ToString(), STATIC_SALT, validator2_acc["address"])
+        vote_hash_1 = get_hash(EXCHANGE_RATES.ToString(), STATIC_SALT, validator1_val["address"])
+        vote_hash_2 = get_hash(EXCHANGE_RATES.ToString(), STATIC_SALT, validator2_val["address"])
 
         wait_for_next_voting_period()
 
         # Submit 1st prevote
-        status = tx_submit_prevote(validator1_acc["name"], vote_hash_1, validator1_home)
+        status, prevote_1 = tx_submit_prevote(validator1_val["name"], vote_hash_1, validator1_home)
```

Comment:
I don't think we need to initialize this here:
```suggestion
        status = tx_submit_prevote(validator1_val["name"], vote_hash_1, validator1_home)
```